contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
```

Comment:
I have a big interest in this as well dealing with secrets. Happy to join in on your discussions @andrewsykim @destijl @jcbsmpsn